Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/delete.go

```diff
@@ -73,6 +82,39 @@ func (f *Factory) NewCmdDelete(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func resourceByLabel(f *Factory, label string) (string, error) {
+	cmdNamespace, err := f.DefaultNamespace()
+	if err != nil {
+		return "", err
+	}
+	kc, err := f.Client()
+	if err != nil {
+		return "", err
+	}
+	selector, err := labels.Parse(label)
+	if err != nil {
+		return "", err
+	}
+	list, err := kc.Events(cmdNamespace).List(selector, fields.Everything())
```

Comment:
> what events are you expecting? i don't think events are created for many things today.

I thought that the addition, modification, or deletion of any kind of resource was an Event.